x-pack/plugins/fleet/public/components/platform_selector.tsx

```diff
@@ -61,10 +63,10 @@ export const PlatformSelector: React.FunctionComponent<Props> = ({
   onCopy,
 }) => {
   const getInitialPlatform = useCallback(() => {
-    if (hasK8sIntegration) return 'kubernetes';
+    if (hasK8sIntegration || typeOfCSPIntegration === 'IS_CSP_KSPM') return 'kubernetes';
```

Comment:
if we pass the whole object, whenever we have an assertion to check if it is CSP, we can do something like that:

```js
if (hasK8sIntegration || cloudSecurityIntegration?.integrationType === FLEET_CLOUD_SECURITY_POSTURE_CSPM_POLICY_TEMPLATE) {
	return 'kubernetes';
} 
```